src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's recommended to style elements using classes to avoid specificity issues and to make styles more reusable. Consider adding a class to the `.stars__star` elements and style them using that class instead.